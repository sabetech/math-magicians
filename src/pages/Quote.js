import React from 'react';

function Quote() {
  return (
    <blockquote>
      Mathematics is not about numbers, equations, computations, or algorithms:
      it is about understanding - William Paul Thurston
    </blockquote>
  );
}

export default Quote;
