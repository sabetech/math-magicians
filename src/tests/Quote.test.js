import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../pages/Quote';

describe('Testing pages that should render correctly', () => {
  it('Quote should render correctly', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
