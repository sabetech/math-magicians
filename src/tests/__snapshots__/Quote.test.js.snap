// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing pages that should render correctly Quote should render correctly 1`] = `
<blockquote>
  Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding - William Paul Thurston
</blockquote>
`;
